specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/apimgatewayConfigConnections.json

```diff
@@ -0,0 +1,344 @@
+          "200": {
+            "description": "Successfully got the API Management gateway config connection resource.",
+            "schema": {
+              "$ref": "#/definitions/ApiManagementGatewayConfigConnectionResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ApiManagementGatewayConfigConnection"
+        ],
+        "operationId": "ApiManagementGatewayConfigConnection_CreateOrUpdate",
+        "description": "Creates or updates an API Management gateway config connection. This is long running operation and could take several minutes to complete.",
+        "x-ms-examples": {
+          "ApiManagementCreateGatewayConfigConnection": {
+            "$ref": "./examples/ApiManagementCreateGatewayConfigConnection.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/GatewayNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ConfigConnectionIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApiManagementGatewayConfigConnectionResource"
+            },
+            "description": "Parameters supplied to the CreateOrUpdate API Management gateway config connection operation."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The gateway config connection was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/ApiManagementGatewayConfigConnectionResource"
+            }
+          },
+          "201": {
+            "description": "The gateway config connection was successfully created.",
+            "schema": {
+              "$ref": "#/definitions/ApiManagementGatewayConfigConnectionResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
Add the `x-ms-long-running-operation-options` property for all occurrences. [https://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options](https://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options)